internal/resources/collection_schema.go

```diff
@@ -0,0 +1,20 @@
+package resources
+
+import (
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema"
+)
+
+func CollectionSchema() schema.Schema {
+	return schema.Schema{
+		Attributes: map[string]schema.Attribute{
+			"organization_id": stringAttribute(required, requiresReplace),
+			"project_id":      stringAttribute(required, requiresReplace),
+			"cluster_id":      stringAttribute(required, requiresReplace),
+			"bucket_id":       stringAttribute(required, requiresReplace),
+			"scope_name":      stringAttribute(required, requiresReplace),
+			"collection_name": stringAttribute(required, requiresReplace),
+			"max_ttl":         int64Attribute(optional),
```

Comment:
If max_ttl is not provided in plan, can it have any computed value, or will it not be returned from the response either?

Reply:
It will have a default value of 0, if its not added to the plan, should this field then have `(optional, computed)` instead?